src/acceptance/acceptance_test.go

```diff
@@ -0,0 +1,59 @@
+package acceptance_test
+
+import (
+	"time"
+
+	"github.com/onsi/gomega/gbytes"
+
+	"os/exec"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("OTel Collector", func() {
+
+	Describe("writes logs to respective files", func() {
+		It("writes resourceSpans", func() {
+			Eventually(func() error {
+				boshCmd := exec.Command("bosh", "ssh", "router/0", "-c", "sudo cat /var/vcap/data/otel-collector/tmp/otel-collector-traces.log | grep -q resourceSpans")
+				boshCmd.Stdout = GinkgoWriter
+				boshCmd.Stderr = GinkgoWriter
+				return boshCmd.Run()
+			}, 60*time.Second, 5*time.Second).Should(Succeed())
+		})
+
+		It("writes resourceMetrics", func() {
+			boshCmd := exec.Command("bosh", "ssh", "diego-cell/0", "-c", "sudo cat /var/vcap/data/otel-collector/tmp/otel-collector-metrics.log | grep -q resourceMetrics")
+			boshCmd.Stdout = GinkgoWriter
+			boshCmd.Stderr = GinkgoWriter
+			Eventually(boshCmd.Run, 60*time.Second, 5*time.Second).Should(Succeed())
```

Comment:
This `Eventually` will not retry the command. Can you fix it up like the others to make it retry?